doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-rest-api-requests-proc.adoc

```diff
@@ -84,6 +84,9 @@ For REST client:
 * *HTTP Headers*: Set the following headers:
 ** `Accept`: `application/json`
 ** `Content-Type`: `application/json`
++
+NOTE: When you add `field=not_null` in the `Content_Type`, then the null fields are excluded from the REST API response.
```

Comment:
```suggestion
NOTE: When you add `field=not_null` to `Content-Type`, the null fields are excluded from the REST API response.
```
"Content-Type" uses hyphens in other instances in this doc, not underscores.
Minimalism - I would remove "then" from "then the" as it is redundant. 
Minimalism - I would replace "in the" with "to" but thats just me.

Reply:
Done, changed it